docker-compose/docker-compose.yml

```diff
@@ -303,7 +303,7 @@ services:
      healthcheck:
        start_period: 60s
        retries: 5
-       test: curl --fail -s http://localhost:46070/api/v1/resource-management/categorization/start/ || exit 1
+       test: curl --fail -s http://localhost:46070/api/v1/resource-management/categorization/run/ || exit 1
```

Comment:
aren't you triggering the categorization with this command?

Reply:
Actually, for doing the health check I created another URL (http://localhost:46070/api/v1/resource-management/categorization/run/)....so, I'm triggering there to check the res-cat is up or down...

Reply:
The name is quite confusing, but the /start endpoint is the one triggering categorization.